app/src/main/java/com/itsaky/androidide/actions/etc/PreviewLayoutAction.kt

```
@@ -1,213 +1,166 @@
+      markInvisible()
+      return
+    }
+
+    val viewModel = data.requireActivity().editorViewModel
+    val editor = data.getEditor()
+    val file = editor?.file
+
+    if (file != null && !viewModel.isInitializing && file.name.endsWith(".xml")) {
+      val type = try {
+        extractPathData(file).type
+      } catch (err: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)